test/unit/core.js

```diff
@@ -1538,3 +1538,19 @@ testIframeWithCallback( "Don't call window.onready (#14802)", "core/onready.html
 			equal( error, false, "no call to user-defined onready" );
 	}
 );
+
+if ( typeof Symbol === "function" && Symbol.iterator ) {
+	test( "Iterability of jQuery objects (gh-1693)", function() {
+		/* jshint unused: false */
+		expect( 1 );
+
+		var i,
+			elem = jQuery( "<div></div><span></span><a></a>" ),
+			result = "";
+
+		try {
+			eval( "for ( i of elem ) { result += i.nodeName; }" );
```

Comment:
OK, but I don't really think that would be easier to write and I prefer to not create globals (even temporary) if that's not necessary.